api/src/main/java/org/openmrs/module/chica/extension/html/AdminList.java

```diff
@@ -31,7 +31,6 @@ public Map<String, String> getLinks() {
 		map.put("module/chica/viewPatient.form", "View Encounters");
 		map.put("module/chica/chicaRuleTester.form", "Rule Tester");
 		map.put("module/chica/chicaNoteTester.form", "Note Tester");
-		map.put("module/chica/cacheConfiguration.form", "Cache Configuration");

```

Comment:
Can you update the version to 1.89.0-SNAPSHOT in all three POM files for the chica module?

Reply:
Sure, let me update the versions in POM files for chica module.